src/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Simplification;
+using Microsoft.DotNet.ApiSymbolExtensions;
+
+namespace Microsoft.DotNet.GenAPI
+{
+    /// <summary>
+    /// Processes assemly symbols to build correspoding structures in C# language.
+    /// Depend on ISyntaxWriter impelemtenation, the result could be C# source file, xml etc.
```

Comment:
Is this line still valid? What ISyntaxWriter is meant here? Also, this class is specific to only emit C#, so xml or other languages don't need to be named here.

Reply:
No, the comment is no longer valid. Updated